src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj

```diff
@@ -14,7 +14,6 @@
     <Reference Include="Microsoft.Bcl.AsyncInterfaces" Condition="'$(TargetFramework)' != '$(DefaultNetCoreTargetFramework)'" />
 
     <Reference Include="Microsoft.Extensions.Features" />
-    <Reference Include="System.IO.Pipelines" />
```

Comment:
This will be needed for netstandard and netfx right?

Reply:
Good point, fixed